cypress/integration/test_chart_of_accounts.js

```diff
@@ -0,0 +1,129 @@
+context('Chart Of Accounts', () => {
+	before(() => {
+		cy.login();
+	});
+
+    beforeEach(() => {
+        cy.visit(`app/account/view/tree`);
+	});
+
+    it('Check if Expand All works', () => {
+        cy.findByRole('button', {name: 'Expand All'}).trigger('click', {force: true});
+        cy.location("pathname").should("eq","/app/account/view/tree");
```

Comment:
this doesn't "check" if "expand all" actually worked, you should check if child items are visible here right?

Reply:
Yeah ideally it should except I don't know of a way right now how this can be done :see_no_evil:

Reply:
check for the existence of `tree-children`?